src/utils/__tests__/common.test.js

```diff
@@ -262,4 +269,36 @@ describe('Common utils', () => {
       expect(parseTokenAmount('100', 0)).toBe(expectedValue);
     });
   });
+
+  describe('getFormattedTransactionFeeValue', () => {
+    it('should parse from BigNumber', () => {
```

Comment:
What happens if `getFormattedTransactionFeeValue` encounters an invalid value altogether? How would it behave?

Reply:
It would throw unhandled exception.